### PR TITLE
Updating more translation files

### PR DESCRIPTION
**Pull request by ThePedroPaulino**
Minor corrections and updates about the ptbr translation keys.

### CODE REVIEW

File: config/locales/ptbr.yml [hunk 1]
```diff
@@ -177,7 +182,7 @@ ptbr:
    dosage: 'dosagem'
    dosage_placeholder: 'e.g. 2mg'
    refill: 'refil'
-   comments_hint: 'Adcione quasiquer comentários adcionais aqui, por exemplo: registrando os sintomas'
+   comments_hint: 'Adcione quaisquer comentários adcionais aqui, por exemplo: Registrando os sintomas'
```
Comment (cpanato) [inline config/locales/ptbr.yml]: typo in the begging: `s/Adcione/Adicione`

File: config/locales/ptbr.yml [hunk 2]
```diff
@@ -8,9 +8,12 @@ ptbr:
  edited_updated_at: '<strong>Data Criação:</strong> %{created_at} <em>(edited %{updated_at})</em>'
  salutation: "Olá %{name},"
  click_here: "clique aqui"
+ less: " [Less]"
```
Comment (cpanato) [inline config/locales/ptbr.yml]: can change here as well: `s/Less/Menos`

File: config/locales/ptbr.yml [hunk 3]
```diff
@@ -74,8 +80,7 @@ ptbr:
   singular: 'Conta'
   sign_in: 'Entrar'
   sign_up: 'Cadastrar'
-  forgot_password: 'Esqueceu sua senha?'
-
+  forgot_password: 'Esqueceu sua senha ?'
```
Comment (cpanato) [inline config/locales/ptbr.yml]: Why a space beteweem `senha` and `? `?

File: Rakefile [hunk 4]
```diff
@@ -25,11 +25,11 @@ end
 desc 'Automate the Config Setup for New Environments'
```
Comment (julianguyen) [inline Rakefile]: Please don't commit this file!

File: config/env/development.env [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# Application
```
Comment (julianguyen) [inline config/env/development.env]: Please don't commit this file!

File: config/env/test.env [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# Application
```
Comment (julianguyen) [inline config/env/test.env]: Please don't commit this file!

File: public/javascripts/i18n.js [hunk 7]
```diff
@@ -12,19 +12,23 @@
 // See tests for specific formatting like numbers and dates.
```
Comment (julianguyen) [inline public/javascripts/i18n.js]: Please don't commit this file! This file gets overridden by the original code automatically.

**Pull request closed without merging**